doc/jsk_pcl_ros/nodes/organized_statistical_outlier_removal.md

```diff
@@ -1,4 +1,5 @@
-# OrganizedStatisticalOutlierRemoval 
+# OrganizedStatisticalOutlierRemoval
```

Comment:
Overall, this PR seems good. One thing I would like to add is the distinction between the usage of Euclidean clustering and this node. Both can be given the same input, and while Euclidean clustering outputs cluster indices, it can ultimately produce a point cloud, making them quite similar in practice. It would be helpful if the documentation could include guidance on how users should differentiate between using this node and Euclidean clustering.
https://github.com/jsk-ros-pkg/jsk_recognition/blob/3ed4aa6c7fdab03c97d208b7021b2f8f2a1104f1/doc/jsk_pcl_ros/nodes/euclidean_clustering.md

Reply:
I intended to use this after Euclidean clustering.
Euclidean clustering makes the clusters with euclidean distance,
and this node can remove the shadow of the points from clusters.
Removing shadow by just using Euclidean clustering is a bit hard, because it does not consider the distribution.

Reply:
OK. Can you write the document in a way that many users can understand? Additionally, I would like you to include an explanation with a link to Euclidean clustering.

Reply:
I added sample launch for this feature, and i think it is enough.
I don't get what you want me to write about it.
Can you clearly tell me what you want me to write?

Reply:
I believe that many of the people who will use this are not developers but rather users.
Among them, few will be able to understand just by looking at the usage of the launch file.

I think many users (including developers) will appreciate it if you could write documentation that even fourth-year undergraduate students can understand.
Could you add and revise the documentation as you answered my questions?

Reply:
Added.